doc/sphinx/source/recipes/recipe_multimodel_products.rst

```diff
@@ -48,7 +48,7 @@ User setting files are stored in recipes/
 Variables
 ---------
 
-* tas(atmos, monthly, longitude, latitude, time)
+* all(atmos, monthly, longitude, latitude, time)
```

Comment:
`all` could be misunderstood for an actual variable name.
What about
```
* any Amon variable (monthly, longitude, latitude, time)
```
?

Reply:
Yes, it is fine